src/main/resources/interface-specification.yml

```diff
@@ -144,6 +144,13 @@ paths:
           type: string
           format: uuid
     get:
+      parameters:
+      - name: riskAnalysisFormCanPassValidation
```

Comment:
are you sure that this was in the requirement?
As far as I understand from the ticket, the new flag is only in the response, not a parameter in the request